packages/core/android/src/oldarch/src/DdLogs.kt

```diff
@@ -0,0 +1,163 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.reactnative
```

Comment:
in this source set all the files should also respect packaging structure and should be placed under `com/datadog/reactnative` folder.